### ISSUE TITLE
Encrypt mail subject / support memoryhole

### ISSUE THREAD

**Issue by rugk** [mailvelope/mailvelope#526]
Other (PGP) addons, notably [TorBirdy](https://blog.torproject.org/blog/torbirdy-023-released), now encrypt the mails Subject and References header.

For the whole specification/idea see http://modernpgp.org/memoryhole/.

**Comment by u451f** [comment]
I'd like to update **and reiterate this request.** 

TorBirdy is now deprecated. However, Thunderbird has their own OpenPGP implementation which supports encrypting the email subject using the above specification called Memoryhole.

However, when sending email from Thunderbird to people using Mailvelope, the encrypted subject does not get decrypted. Instead, they see "…". This is a bit annoying as then people also reply with this email subject and it's hard to follow up on discussions. This has for example also been reported in this comment: https://github.com/mailvelope/mailvelope/issues/762#issuecomment-856740838
